### PR TITLE
[#157588544] WIP icon font integration

### PR DESCRIPTION
**Pull request by ClaudioConversa**
create a wrapper to connect react-native-vector-icons/Icon component with the native-base StyleProvider , applied to all icons in the project.
the first icon in the landing page on "Login with SPID" button need to be update when we the right icon will implemented into the font

screenshot_1528973873
screenshot_1528973840
screenshot_1528973807
screenshot_1528973802
screen shot 2018-06-14 at 13 07 14

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cloudify** [comment]
@ClaudioConversa thanks, could you post a couple screenshots? thanks

File: ts/screens/wallet/AddPaymentMethodScreen.tsx [hunk 1]
```diff
@@ -147,7 +147,7 @@ export class AddPaymentMethodScreen extends React.Component<Props, State> {
                   </Grid>
                 </Left>
                 <Right style={AddMethodStyle.centeredContents}>
-                  <Icon
+                  <IconFont
                     name={itemInfo.item.icon}
                     color={variables.brandPrimary}
```
Comment (fpersico) [inline ts/screens/wallet/AddPaymentMethodScreen.tsx]: Does this color (and also size) property take precedence over the default value of the theme?

Comment (ClaudioConversa) [reply ts/screens/wallet/AddPaymentMethodScreen.tsx]: no they doesn't

Comment (fpersico) [reply ts/screens/wallet/AddPaymentMethodScreen.tsx]: So what are they useful for?

Comment (ClaudioConversa) [reply ts/screens/wallet/AddPaymentMethodScreen.tsx]: they was converted before connection with the style, i will fix it in the next commit

**Pull request merged**